digital_land/commands.py

```diff
@@ -140,8 +140,16 @@ def pipeline_run(
     default_values = pipeline.default_values(endpoints=endpoints)
     combine_fields = pipeline.combine_fields(endpoints=endpoints)
 
+    lpa_geometry_path = "var/cache/local-authority.csv"
+    local_authority_path = "var/cache/local-authority.csv"
```

Comment:
I've hardcoded these for now, it should be pretty easy to add these as inputs to the pipeline run command, but I'm not sure where that run command is actually triggered